File: App.API/Models/RestaurantOrder.cs
using System;
using System.Collections.Generic;

namespace App.API.Models
{
    public class RestaurantOrder
    {
        public int Id { get; set; }
        public string OrderFor { get; set; }
        public DateTime OrderDate { get; set; }
        public bool IsCompleted { get; set; }
        public bool IsPaid { get; set; }
        public decimal TotalPriceBeforeTax { get; set; }
        public decimal TaxAmount { get; set; }
        public decimal TotalPrice { get; set; }
        public ICollection<RestaurantOrderItem> RestaurantOrderItems { get; set; }
    }
}

File: App-SPA/src/app/_models/restaurantOrder.ts
import { RestaurantOrderItem } from './restaurantOrderItem';

export interface RestaurantOrder {
    id: number;
    bookingId: number;
    orderFor: string;
    orderDate: Date;
    isCompleted: boolean;
    isPaid: boolean;
    totalPriceBeforeTax: number;
    taxAmount: number;
    totalPrice: number;
    restaurantOrderItems: RestaurantOrderItem[];
}
